pkg/client/shub/pull.go

```diff
@@ -41,12 +41,6 @@ func DownloadImage(filePath string, shubRef string, force, noHTTPS bool) (err er
 		sylog.Infof("Download filename not provided. Downloading to: %s\n", filePath)
 	}
 
-	if !force {
```

Comment:
We should either keep this check, or remove the `force` parameter from the function call so that it doesnt silently do nothing

Reply:
Oh yeah, I'll put the check back...